app/views/questionnaire.py

```diff
@@ -170,13 +176,15 @@ def post_household_composition(routing_path, **kwargs):
     return _render_page(routing_path, block, current_location, post_form=form)
 
 
-@questionnaire_blueprint.route('thank-you', methods=['GET'])
-def get_thank_you(eq_id, form_type, collection_id):  # pylint: disable=unused-argument
-    survey_completed_metadata = current_app.eq['session_storage'].get_survey_completed_metadata()
-    schema = load_schema_from_params(eq_id, form_type)
+@after_submission_blueprint.route('/thank-you', methods=['GET'])
+def get_thank_you():  # pylint: disable=unused-argument
+    session_data = current_app.eq['session_storage'].get_session_data()
+    eq_id, form_type = session_data.get('schema_identifier').split('_', maxsplit=1)
+    metadata = get_metadata(current_user)
```

Comment:
Here we call `current_app.eq['session_storage'].get_session_data()` which queries `EQSession` and then 2 lines later we call `get_metadata(current_user)` which queries `EQSession` again.
Is there a way we can limit this to 1 call per request?